RecoVertex/Configuration/python/RecoVertex_cff.py

```diff
@@ -36,33 +36,78 @@
 #timing
 from RecoVertex.PrimaryVertexProducer.TkClusParameters_cff import DA2D_vectParameters
 DA2D_vectParameters.TkDAClusParameters.verbose = cms.untracked.bool(False)
-unsortedOfflinePrimaryVertices4D = unsortedOfflinePrimaryVertices.clone( verbose = cms.untracked.bool(False),
+unsortedOfflinePrimaryVertices4DnoPID = unsortedOfflinePrimaryVertices.clone( verbose = cms.untracked.bool(False),
                                                                          TkClusParameters = DA2D_vectParameters )
-unsortedOfflinePrimaryVertices4D.TkFilterParameters.minPt = cms.double(0.0)
-unsortedOfflinePrimaryVertices4D.TrackTimesLabel = cms.InputTag("trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModel")
-unsortedOfflinePrimaryVertices4D.TrackTimeResosLabel = cms.InputTag("trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModelResolution")
-offlinePrimaryVertices4D=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4D", particles="trackRefsForJetsBeforeSorting4D", trackTimeTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModel"),trackTimeResoTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModelResolution"),assignment=dict(useTiming=True))
-offlinePrimaryVertices4DWithBS=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4D:WithBS", particles="trackRefsForJetsBeforeSorting4D", trackTimeTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModel"),trackTimeResoTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModelResolution"),assignment=dict(useTiming=True))
+unsortedOfflinePrimaryVertices4DnoPID.TkFilterParameters.minPt = cms.double(0.0)
+unsortedOfflinePrimaryVertices4DnoPID.TrackTimesLabel = cms.InputTag("trackExtenderWithMTD:generalTrackt0")
+unsortedOfflinePrimaryVertices4DnoPID.TrackTimeResosLabel = cms.InputTag("trackExtenderWithMTD:generalTracksigmat0")
+trackWithVertexRefSelectorBeforeSorting4DnoPID = trackWithVertexRefSelector.clone(vertexTag="unsortedOfflinePrimaryVertices4DnoPID")
+trackWithVertexRefSelectorBeforeSorting4DnoPID.ptMax=9e99
```

Comment:
parameters can be changed within the clone statement, rather than afterward (reduce repetition of module name)